nodes/node_crate.lua

```diff
@@ -294,9 +299,9 @@ minetest.register_node("digtron:loaded_crate", {
 	on_receive_fields = function(pos, formname, fields, sender)
 		return loaded_on_recieve(pos, fields, sender)
 	end,
-		
+
 	on_dig = function(pos, node, player)
-		return loaded_on_dig(pos, player, "digtron:loaded_crate")
+		return player and not minetest.is_protected(pos, player:get_player_name()) and loaded_on_dig(pos, player, "digtron:loaded_crate")
```

Comment:
Please don’t mix action (`loaded_on_dig`) and tests in single expression. Use `if...then` instead, as in protected crate code.